### ISSUE TITLE
enable full functionality like on screenshot - Cartalyst Sentinel

### ISSUE THREAD

**Issue by ghost** [vsch/laravel-translation-manager#75]
Hi

Im new to laravel and cant seem to get this package working. The basic edit functionality exists but I cant see the publish options which an admin I think should have. 

I have followed the installation steps very carefully but I think I miss some key laravel knowledge. In my routes.php I have the following: 

```
use Vsch\TranslationManager\Translator;
\Route::group(['middleware' => 'web', 'prefix' => 'translations'], function () {
     Translator::routes();
 });

```
But Im not sure as to what the 'web' middleware needs to do. Currently it looks like it is going through some vanilla laravel middlewares. If I replace the web middleware what then should it do? 

I have also defined the $gate variables as per instruction but cant really see the effect. Does anyone maybe have an example of a working example? 

Thanks.

**Comment by ghost** [comment]
Is there a way to show the full web interface without logging in?

**Comment by vsch** [comment]
@highnoon112, how to allow administration of translations is described in the wiki https://github.com/vsch/laravel-translation-manager/wiki/Installation#setting-up-user-authorization

Showing the full interface without logging in is not supported but can probably be hacked but why would you want to allow anyone to modify the translations?

**Comment by ghost** [comment]
@vsch, you are right. It doesnt make sense. I just cant get it to work and thought Id start from a point where I know it works and gradually built in the user authorisation. 

Ive gone through the link you mentioned multiple times but Im missing some key laravel concepts I think. 

My project is authenticating through Sentinel package, from Cartalyst. Ive added the following code to AuthServiceProvider::boot but without result. 
```
$gate->define('ltm-admin-translations', function ($user) {
     return true;
 });
```
To be honest Im not sure what Im doing wrong. Im not sure either how LTM would check whether the user has this access rights.

**Comment by ghost** [comment]
@vsch thanks for reflecting. 

Got it working. No idea how. xD

**Comment by nohumans** [comment]
Hi!
This is may working code on 5.4


```
   /**
     * Register any authentication / authorization services.
     *
     * @return void
     */
    public function boot()
    {
        $this->registerPolicies();
        // Traductor
        Gate::define('ltm-admin-translations', function ($user) {
            return $user->isSuperAdmin;
        });

        Gate::define('ltm-bypass-lottery', function ($user) {
            return $user && ($user->isSuperAdmin || $user->role == 'traductor');
        });

        //
        Gate::define('ltm-list-editors', function ($user, $connection_name, &$user_list) {
            $query = $user->on($connection_name)->getQuery();

            $user_list = $query->orderby('id')->where('role','traductor')->get(['id', 'email']);

            // if the returned list is empty then no per locale admin will be shown for
            // the current user.
            return $user_list;
        });
 }
```
